benchmarks/src/tpch.rs

```diff
@@ -147,155 +135,6 @@ pub fn get_tpch_table_schema(table: &str) -> Schema {
     }
 }
 
-/// Get the expected schema for the results of a query
```

Comment:
this code is not used it seems. However,  clippy didn't complain as it was declared `pub`.

https://github.com/search?q=repo%3Aapache%2Farrow-datafusion%20get_answer_schema&type=code